languages/java/exercises/concept/conditionals-ternary/.docs/introduction.md

```diff
@@ -0,0 +1,28 @@
+## Ternary Operator
+
+The _ternary operator_ is a lightweight, compact alternative for simple _if/else_ statements. Usually used in (but not restricted to) return statements, it needs just one single to make the decision, returning the left value if the expression is `true` and the right value if `false`, as follows:
+
+```java
+
+boolean expr = 0 != 200;
+
+int value = expr ? 22 : 33;
+// => 22
+
+```
+
+A lot of simple _if/else_ expressions can be simplified using _ternary operators_
+
+```java
+
+// this ternary statement
+return 5 > 4 ? true : false;
+
+// is equivalent to this if/else
+if ( 5 > 4 ) {
+    return true;
+} else {
+    return false;
+}
+
+```
```

Comment:
One of our goals with the `introduction.md` document is to have them be very concise (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsintroductionmd)). As the above example introduces the ternary statement syntax, this example can be removed:

```suggestion
```